src/test/java/Homework19a.java

```diff
@@ -0,0 +1,47 @@
+import net.bytebuddy.asm.Advice;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public  class Homework19a extends BaseTest{
+
+
+    @Test
+    public void DeleteAPlaylistTest() throws InterruptedException{
+        String playlistName = "Timelaps";
+
+
+        login("demo@class.com","te$t$tudent");
+        wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("i[@title='Create a new playlist'])");
```

Comment:
Here is some mess with braces
It should be like this
wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("i[@title='Create a new playlist'])")));